pages/reference/OS/updates/self-service.md

```diff
@@ -33,6 +33,11 @@ Update time can vary significantly, depending on the speed of your network, the
 
 If your {{ $names.os.lower }} update fails for any reason, the device should still be recoverable. For devices running {{ $names.os.lower }} 2.x, the user application should still be running normally, and the reasons for failure can be examined through the update logs at `/mnt/data/balenahup/` (or potentially the legacy location `/mnt/data/resinhup/`). If you have any issues, please contact us on the [troubleshooting section of the forums][troubleshooting].
 
+## Update locks
+
+[Update locks](https://docs.balena.io/learn/deploy/release-strategy/update-locking/) is a mechanism that allows applications to enter critical sections and prevent updates that would interrupt the application from running. Update locks can also be used to delay the reboot that applies a hostOS update operation until the application exists the critical section by removing the update locks. [Overriding update locks](https://docs.balena.io/learn/deploy/release-strategy/update-locking/#overriding-the-lock) will ignore existing locks and allow a hostOS update process to proceed with a reboot.
```

Comment:
sorry @jaomaloy I might need to be spell checked :)

s/sections and prevent updates/ections and prevents updates

Reply:
also 

s/application exists/application exits

Reply:
The second correction makes sense but the first correction doesn't seem necessary to me. @alexgg can you check it now?

Reply:
> The second correction makes sense but the first correction doesn't seem necessary to me. @alexgg can you check it now?

hey @jaomaloy not a native speaker either, but now we are using different tenses:

> a mechanism that **allows** applications ... and **prevent** updates